src/components/tabs/tabs.component.js

```diff
@@ -131,6 +131,32 @@ const Tabs = ({
     [onTabChange, isTabSelected, hasTabStop]
   );
 
+  const blurPreviousSelectedTab = useCallback(() => {
+    const previousTabIndex = tabIds.indexOf(previousSelectedTabId.current);
+    /* istanbul ignore else */
+    if (previousTabIndex !== -1) {
+      const previousTabRef = tabRefs[previousTabIndex];
+      previousTabRef.current?.blur();
+    }
+  }, [tabIds, tabRefs]);
+
+  useEffect(() => {
+    if (previousSelectedTabId.current !== selectedTabId) {
+      if (selectedTabId !== selectedTabIdState) {
+        setSelectedTabIdState(selectedTabId);
+        blurPreviousSelectedTab();
+      }
+      previousSelectedTabId.current = selectedTabId;
+    }
+  }, [
+    blurPreviousSelectedTab,
+    previousSelectedTabId,
+    selectedTabId,
+    selectedTabIdState,
+    tabIds,
```

Comment:
Does `tabIds` and `tabRefs` need to be in the dependency array if `blurPreviousSelectedTab` is?

Reply:
Agree with Ed here. The values are already referenced inside the callback dependency array.